dispatcher-consumer/handler.go

```diff
@@ -1,15 +1,88 @@
 package dispatcherconsumer
 
 import (
+	"config-manager/domain/message"
+	kafkaUtils "config-manager/infrastructure/kafka"
 	"context"
+	"encoding/json"
 	"fmt"
+	"log"
+
+	"github.com/google/uuid"
 
 	kafka "github.com/segmentio/kafka-go"
 )
 
 type handler struct {
+	producer      kafkaUtils.KafkaWriterInterface
+	uuidGenerator func() uuid.UUID
+}
+
+// This message to inventory is constructed using data from provided labels. This will change
+// once this information can be consumed from the run_hosts topic.
+func buildMessage(payload message.DispatcherEventPayload, reqID uuid.UUID) ([]byte, error) {
+	msg := message.InventoryUpdate{
+		Operation: "add_host",
```

Comment:
Is `add_host` really the operation we're supposed to use? AFAIK the system-profile inventory consumer does everything but adding new hosts 🤯

Reply:
Yep. They're discussing enforcing a different operation type, but currently `add_host` is to be used.